### PR TITLE
ENG-4685 add to the operator the capability to select the tomcat image for entando-de-app

### PR DESCRIPTION
**Pull request by entgigi**

### CODE REVIEW

File: src/main/resources/crd/entandoapps.entando.org.crd.yaml [hunk 1]
```diff
@@ -233,14 +233,16 @@ spec:
               description: >
                 Specifies the standard server base image that this EntandoApp has been packaged in. Valid values
                 are wildfly (https://github.com/entando/entando-docker-base-images/tree/master/entando-wildfly17-base)
```
Comment (zonia3000) [inline src/main/resources/crd/entandoapps.entando.org.crd.yaml]: can we update also this link?

File: src/main/resources/crd/entandoapps.entando.org.crd.yaml [hunk 2]
```diff
@@ -233,14 +233,16 @@ spec:
...
+                or tomcat (https://github.com/entando/entando-app-engine-base/tree/develop/tomcat)
                 or eap (https://github.com/entando/entando-docker-base-images/tree/master/entando-eap72-clustered-base).
```
Comment (zonia3000) [inline src/main/resources/crd/entandoapps.entando.org.crd.yaml]: can we update also this link?

Comment (entgigi) [reply src/main/resources/crd/entandoapps.entando.org.crd.yaml]: done

File: src/main/resources/crd/entandoapps.entando.org.crd.yaml (hunk 1 above)
Comment (entgigi) [reply src/main/resources/crd/entandoapps.entando.org.crd.yaml]: done

**Review by zonia3000 — APPROVED** [review]

**Review by firegloves — APPROVED** [review]
LGTM

**Pull request merged**